### PR TITLE
[TAS-889] Add related link to likerland 

### PR DESCRIPTION
**Pull request by AuroraHuang22**
ISCN is not preminted
截圖 2023-12-28 晚上11 37 09

ISCN is preminted
截圖 2023-12-28 晚上11 42 17

Search for owner
截圖 2023-12-29 凌晨12 30 29

### CODE REVIEW

File: pages/view/_iscnId/index.vue [hunk 1]
```diff
@@ -111,6 +112,19 @@
         :to="localeLocation({ name: 'nft-iscn-iscnId', params: { iscnId: iscnId }, query: mintQueries })"
         :text="$t('NFTPortal.button.mint')"
       />
+      <div class="flex justify-center items-center p-[4px] rounded-[12px] border-like-cyan-light border-[2px]">
+        <Button
+          v-if="!isShowMintButton"
```
Comment (williamchong) [inline pages/view/_iscnId/index.vue:117]: v-if=!!classId seems to be a more straight forward condition

File: pages/search/_keyword.vue [hunk 2]
```diff
@@ -54,11 +54,24 @@
...
+        <Label
+          class="text-center text-medium-gray"
+          preset="p5"
+          align="center"
+          :text="searchResultText"
+        />
+        <a
+          v-if="owner"
+          :href="ownerPortfolioUrl"
+          target="_blank"
```
Comment (williamchong) [inline pages/search/_keyword.vue:71]: Please add noopener

**Review by williamchong — CHANGES_REQUESTED** [review]

File: pages/view/_iscnId/index.vue (hunk 1 above)
Comment (AuroraHuang22) [reply pages/view/_iscnId/index.vue]: [4f527be](https://github.com/likecoin/app-like-co/pull/430/commits/4f527bed4c22b149766cd653e6b41851851bf034)

File: pages/search/_keyword.vue (hunk 2 above)
Comment (AuroraHuang22) [reply pages/search/_keyword.vue:71]: [741ccc9](https://github.com/likecoin/app-like-co/pull/430/commits/741ccc94574dc2f6619b7b43b9cd0f43bbcc6ffe)

**Review by williamchong — APPROVED** [review]

**Pull request merged**